src/edge/models/chunk.py

```diff
@@ -300,6 +300,14 @@ class Meta:
     def set_qualifiers(self, qualifiers):
         self._qualifiers = json.dumps(qualifiers)
 
+    @property
+    def sequence(self):
```

Comment:
any thoughts wrt strand here? If we are returning the + strand template, and the feature is on the - strand, how will this be returned?

Reply:
This will always return the sequence in the + strand (so if you want to get the actual nucleotides for a - strand feature, you will have to reverse complement the `.sequence` property).

Reply:
Oh i see what you mean. May be better to return the actual (correct orientation) sequence since that is what is important.